simSPI/tests_simple_simulator/test_ctf.py

```diff
@@ -0,0 +1,18 @@
+"""Test function for CTF module."""
+from ..linear_simulator.modules import CTF
+from ..linear_simulator.utils import init_data, normalized_mse, primal_to_fourier_2D
+
+
+def test_ctf():
+    """Test accuracy of the ctf."""
+    path = "simSPI/tests_simple_simulator/tests_data.npy"
```

Comment:
shouldn't you re-generate this data? if the ctf code is changed, and things are consistent, but the new output doesn't correspond to `tests_data.npy`, wouldn't this test fail?

Reply:
same goes for other tests using this .npy file (Noise, Projector, Shift, etc)

Reply:
One could argue this would allow to detect when the CTF code changes and act accordingly.

Reply:
I agree with @fredericpoitevin It is much more secure for future.